README.md

```diff
@@ -62,6 +62,16 @@ Mastodon acts as an OAuth2 provider, so 3rd party apps can use the REST and Stre
 
 ## Deployment
 
+### Getting Started with GitHub Codespaces
```

Comment:
I'm uncertain whether we want to mention this in the README, but if we do, it certainly shouldn't be in the `Deployment` section. Certainly nobody is going to be using Codespaces to run production mastodon servers.

Reply:
Probably it would make the most sense to mention this below the Vagrant section below.

Reply:
Makes sense, moved.